src/main/resources/kafka.properties

```diff
@@ -0,0 +1,6 @@
+bootstrap.servers=localhost:9092
```

Comment:
unnecessary file
all this options should be set via the configuration

Reply:
properties file deleted